### PR TITLE
Collection of fixes

### PR DESCRIPTION
**Pull request by neutmute**
Hi @KennethB  - Apologies for the massive PR however I thought you had abandoned the repo as you hadn't been responding to any issues/chocolatey owner contacts/disqus feeds so I went and started my own fork and made lots of updates.

Now that you are back in the game, here are my collection of updates for your perusal.
Let me know what you think

From my repo: https://github.com/neutmute/chocolatey-vs

Addresses issues #3 and #2

### CODE REVIEW

**Comment by neutmute** [comment]
In comparing the diff's I may have messed up the `admin.xml` files. Let me check and fix if required

File: VisualStudio2015Community/Tools/AdminDeployment.xml [hunk 1]
```diff
@@ -27,24 +37,21 @@
...
-    <SelectableItemCustomization Id="PowerShellToolsV1" Hidden="no" Selected="no" FriendlyName="PowerShell Tools for Visual Studio" />
     <SelectableItemCustomization Id="SilverLight_Developer_KitV1" Hidden="no" Selected="no" FriendlyName="Silverlight Development Kit" />
-    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV9" Hidden="no" Selected="no" FriendlyName="Tools (1.3.1) and Windows 10 SDK (10.0.10586)" />
+    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV10" Hidden="no" Selected="no" FriendlyName="Tools (1.3.2) and Windows 10 SDK (10.0.10586)" />
     <SelectableItemCustomization Id="Win10_EmulatorV1" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10240)" />
     <SelectableItemCustomization Id="Win10_EmulatorV2" Hidden="no" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10586)" />
-    <SelectableItemCustomization Id="XamarinVSCoreV2" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.3)" />
+    <SelectableItemCustomization Id="XamarinVSCoreV3" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.4)" />
```
Comment (neutmute) [inline VisualStudio2015Community/Tools/AdminDeployment.xml:52]: This is the newer admin file - notice `XamarinVSCoreV3` 4.0.4 vs 4.0.3 and above `Windows10_ToolsAndSDKV10` 1.3.2

File: VisualStudio2015Enterprise/Tools/AdminDeployment.xml [hunk 2]
```diff
@@ -30,26 +40,22 @@
...
-    <SelectableItemCustomization Id="PowerShellToolsV1" Hidden="no" Selected="no" FriendlyName="PowerShell Tools for Visual Studio" />
     <SelectableItemCustomization Id="SilverLight_Developer_KitV1" Hidden="no" Selected="no" FriendlyName="Silverlight Development Kit" />
-    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV9" Hidden="no" Selected="no" FriendlyName="Tools (1.3.1) and Windows 10 SDK (10.0.10586)" />
+    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV10" Hidden="no" Selected="no" FriendlyName="Tools (1.3.2) and Windows 10 SDK (10.0.10586)" />
     <SelectableItemCustomization Id="Win10_EmulatorV1" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10240)" />
     <SelectableItemCustomization Id="Win10_EmulatorV2" Hidden="no" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10586)" />
-    <SelectableItemCustomization Id="XamarinVSCoreV2" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.3)" />
+    <SelectableItemCustomization Id="XamarinVSCoreV3" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.4)" />
```
Comment (neutmute) [inline VisualStudio2015Enterprise/Tools/AdminDeployment.xml:56]: This is the newer admin file - `XamarinVSCoreV3` vs `XamarinVSCoreV2`

File: VisualStudio2015Professional/Tools/AdminDeployment.xml [hunk 3]
```diff
@@ -20,34 +29,29 @@
...
-    <SelectableItemCustomization Id="PowerShellToolsV1" Hidden="no" Selected="no" FriendlyName="PowerShell Tools for Visual Studio" />
     <SelectableItemCustomization Id="SilverLight_Developer_KitV1" Hidden="no" Selected="no" FriendlyName="Silverlight Development Kit" />
-    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV9" Hidden="no" Selected="no" FriendlyName="Tools (1.3.1) and Windows 10 SDK (10.0.10586)" />
+    <SelectableItemCustomization Id="Windows10_ToolsAndSDKV10" Hidden="no" Selected="no" FriendlyName="Tools (1.3.2) and Windows 10 SDK (10.0.10586)" />
     <SelectableItemCustomization Id="Win10_EmulatorV1" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10240)" />
     <SelectableItemCustomization Id="Win10_EmulatorV2" Hidden="no" Selected="no" FriendlyName="Emulators for Windows 10 Mobile (10.0.10586)" />
-    <SelectableItemCustomization Id="XamarinVSCoreV2" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.3)" />
+    <SelectableItemCustomization Id="XamarinVSCoreV3" Hidden="no" Selected="no" FriendlyName="C#/.NET (Xamarin v4.0.4)" />
```
Comment (neutmute) [inline VisualStudio2015Professional/Tools/AdminDeployment.xml:52]: newer admin file `XamarinVSCoreV3` vs `XamarinVSCoreV2`

**Comment by neutmute** [comment]
After checking the file diffs - I think mine are the newer ones. Not sure how they got so different? 
I generated mine using `vs_enterprise.exe  /CreateAdminFile "D:\downloads\vsent.xml"` etc

**Comment by KennethB** [comment]
Yes looks like they've done some small updates

**Pull request merged**